xscen/aggregate.py

```diff
@@ -580,7 +585,10 @@ def produce_horizon(
             )
 
         else:
-            ds_mean = ds_mean.swap_dims({"time": "horizon"}).drop_vars("time")
+            ds_mean = ds_mean.expand_dims(
+                dim={"horizon": [f"{ds.time.dt.year[0]}-{ds.time.dt.year[-1]}"]}
```

Comment:
isn't the point that there is no `time` dimension when freq== fx ?

Reply:
My use case here is indicators in xclim that compute return periods. They collapse `time` by construction and are thus given `freq=='fx'`, but they are still associated to a specific time period (i.e. 100-year maximum streamflow for 1981-2010).